puppet/modules/scaleway/manifests/init.pp

```diff
@@ -0,0 +1,25 @@
+# Class to deploy snapshot management for Scaleway hosts
+#
+class scaleway(
+  Hash[String, Hash] $servers = {},
+) {
+
+  # To hold the API config
+  file { '/etc/scaleway':
+    ensure => 'directory',
+    owner  => 'root',
+    group  => 'root',
+    mode   => 0700,
+  }
+
+  file { '/usr/bin/scaleway-snapshot':
+    ensure => $ensure,
```

Comment:
I don't think `$ensure` is defined here.

Reply:
dammit. Added an Enum for that :)